.github/workflows/post-merge-postoffice-mock-to-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy PostOffice Mock to Build
+on:
+  workflow_dispatch:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches:
+      - main
+    paths:
+      - 'post-office-stub/**'
```

Comment:
Do we want to [exclude this path from the main builds ](https://github.com/govuk-one-login/ipv-cri-f2f-api/blob/KIWI-948b/.github/workflows/post-merge-package-to-dev.yml#L12C1-L12C1)so that they're not run for changes to the postoffice mock?

Reply:
Done thanks